README.md

```diff
@@ -7,20 +7,70 @@
 Free SVG icon font for popular brands. See them all on one page at <a href="https://simpleicons.org">SimpleIcons.org</a>. Contributions, corrections & requests can be made on GitHub. Started by <a href="https://twitter.com/bathtype">Dan Leech</a>.</p>
 </p>
 
-## Usage
+## Setup
+
+### From CDN
+
+```html
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet"
+          href="https://cdn.jsdelivr.net/npm/simple-icons-font/font/simple-icons.min.css"
+          type="text/css">
+  </head>
+  <body>
+    <i class="simpleicons simpleicons-simpleicons simpleicons--color"></i>
+  </body>
+</html>
+```
 
-### General Usage
+### Manual download
+
+1. Download the latest version of the package from
+[here][npm-registry-tarball-link], extract it and move the content of the
+`font` folder to your desired location. 
```

Comment:
Could you keep the formatting style that is used in the main repository and write this as a single line? Also, using the text "here" for a link is bad for accessibility and often unnecessary, e.g here you can just do

```suggestion
1. [Download the latest version of the package][npm-registry-tarball-link], extract it, and move the content of the `font` folder to your desired location.
```

<sup>Alternatively, since this is technically invalid MarkDown, the two alternative lines should be indented with 3 spaces to line up properly.</sup>